include/matx/core/allocator.h

```diff
@@ -413,6 +413,15 @@ struct matx_allocator {
   }  
 };
 
-
+inline std::string SpaceString(matxMemorySpace_t space) {
```

Comment:
This should be `__MATX_INLINE__`

Reply:
Changed (but please note that most methods in this file still use "inline", should that change too?)

Reply:
Yes, we try to use force inline in most places, so please change to the macro instead

Reply:
done!